test/module/irohad/model/converters/json_transaction_test.cpp

```diff
@@ -48,3 +52,41 @@ TEST_F(JsonTransactionTest, InvalidWhenFieldsMissing){
 
   ASSERT_FALSE(serial_transaction.has_value());
 }
+
+TEST_F(JsonTransactionTest, InvalidWhenNegativeAddAssetQuantity) {
+  auto transaction_string = "{\n"
+      "    \"signatures\": [\n"
```

Comment:
I have a feeling that it is possible to simplify with raw strings (https://solarianprogrammer.com/2011/10/16/cpp-11-raw-strings-literals-tutorial/)